<launch>
  <!-- Start GPS to Odom node -->
  <node  pkg="first_project" name="gps_to_odom" type="gps_to_odom" output="screen">
    <param name="lat_r" value="169.039"/>
    <param name="lon_r" value="45.478"/>
    <param name="alt_r" value="9.227"/>
  </node> <!--these data are taken as the first value sensor_msgs/NavSatFix of topic /fix (GPS data) observed via the plotjuggler -->

  <!-- Start Odom to TF node for wheel odometry -->
  <node pkg="first_project" name="wheel_odom_to_tf" type="odom_to_tf" output="screen">
    <remap from="input_odom" to="odom"/>
    <param name="root_frame" value="world"/>
    <param name="child_frame" value="wheel_odom"/>
  </node>

  <!-- Start Odom to TF node for GPS odometry -->
  <node pkg="first_project" name="gps_odom_to_tf" type="odom_to_tf" output="screen">
    <remap from="input_odom" to="gps_odom"/>
    <param name="root_frame" value="world"/>
    <param name="child_frame" value="gps_odom"/>
  </node>

  <node pkg="first_project" name="lidar_visualization_node" type="lidar_visualization_node" output="screen" />
  <node name="rviz" pkg="rviz" type="rviz" />

</launch>